qa/evil-tests/src/test/java/org/elasticsearch/tribe/TribeUnitTests.java

```diff
@@ -54,13 +54,12 @@
     @BeforeClass
     public static void createTribes() {
         Settings baseSettings = Settings.builder()
-            .put(InternalSettingsPreparer.IGNORE_SYSTEM_PROPERTIES_SETTING, true)
             .put("http.enabled", false)
             .put("node.mode", NODE_MODE)
             .put("path.home", createTempDir()).build();
 
-        tribe1 = NodeBuilder.nodeBuilder().settings(Settings.builder().put(baseSettings).put("cluster.name", "tribe1").put("node.name", "tribe1_node")).node();
-        tribe2 = NodeBuilder.nodeBuilder().settings(Settings.builder().put(baseSettings).put("cluster.name", "tribe2").put("node.name", "tribe2_node")).node();
+        tribe1 = new TribeClientNode(Settings.builder().put(baseSettings).put("cluster.name", "tribe1").put("name", "tribe1_node").build()).start();
+        tribe2 = new TribeClientNode(Settings.builder().put(baseSettings).put("cluster.name", "tribe2").put("name", "tribe2_node").build()).start();
     }
```

Comment:
agreed that this test is way too delicate, testing the tribe node is not easy. We would need a proper qa test for that with external nodes etc. `TribeUnitTests` should have failed but it didn't, because we specify the config using `path.conf`, which is not passed to the tribe clients, so in our unrealistic test the tribe clients don't actually load the same config because they don't know where it is, and that makes the test succeed. I tried making the test fail by writing our own config file within temp_dir/config but that didn't quite work either (meaning that the test still succeeds). I think too many changes have happened since the test was written, we should get rid of it and have a proper qa test with external nodes.